src/Hosting/TestHost/src/TestServerOptions.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.TestHost
+{
+    /// <summary>
+    /// Options for the test server.
+    /// </summary>
+    public class TestServerOptions
```

Comment:
Please add BaseAddress also.
https://github.com/dotnet/aspnetcore/blob/39c471a4a74aeadd4f3157f0556bf0bd45d0d947/src/Hosting/TestHost/src/TestServer.cs#L78

Reply:
It is done.